src/utils/beneficial.owner.gov.ts

```diff
@@ -127,6 +129,8 @@ const setBeneficialOwnerData = (reqBody: any, id: string): ApplicationDataType =
     : {};
   data[StartDateKey] = mapFieldsToDataObject(reqBody, StartDateKeys, InputDateKeys);
 
+  data[CeasedDateKey] = reqBody["is_still_bo"] === '0' ? mapFieldsToDataObject(reqBody, CeasedDateKeys, InputDateKeys) : {};
```

Comment:
Where is the test for this logic change?

Reply:
It was reverted from the ceased date fields from the add new BO / MO pages and it's just being put back here.